charts/app-store/templates/backend/backend-service.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: appstore-backend
+spec:
+  selector:
+    app: appstore-backend
+  ports:
+    - name: http
+      port: {{ .Values.appstore_backend.port }}
```

Comment:
in the `values.yaml` file shouldn't `appstore_backend` be just `backend` to match the value?